### PR TITLE
Add possibility to change node address

### PR DESCRIPTION
**Pull request by aderrod94**
Resolves # .

-
-
-

@reviewer-or-team

### CODE REVIEW

File: website/frontend/src/api/initApi.ts [hunk 1]
```diff
@@ -25,6 +25,12 @@ class NodeApi {
   }
 
   async init() {
+    const addressFromLocalStorage = window.localStorage.getItem('node_address');
+
+    if (addressFromLocalStorage) {
+      this.address = addressFromLocalStorage;
+    }
+
```
Comment (osipov-mit) [inline website/frontend/src/api/initApi.ts:33]:
```suggestion
    this.address = window.localStorage.getItem('node_address') || this.address;
```
What do you think?

File: website/frontend/src/components/blocks/Header/Header.tsx [hunk 2]
```diff
@@ -24,14 +30,44 @@ export const Header: VFC = () => {
...
+        id: 1,
+        caption: 'test network',
+        nodes: [{ id: 1, isChoose: false, address: 'wss://rpc-node.gear-tech.io:443' }],
+      },
+      {
+        id: 2,
+        caption: 'development',
+        nodes: [
+          { id: 2, isChoose: false, address: 'ws://localhost:9944' },
+          { id: 3, isChoose: false, address: 'ws://13.56.230.75:9944' },
```
Comment (osipov-mit) [inline website/frontend/src/components/blocks/Header/Header.tsx:51]: this address isn't necessary here

**Review by osipov-mit — CHANGES_REQUESTED** [review]

File: website/frontend/src/api/initApi.ts (hunk 1 above)
Comment (aderrod94) [reply website/frontend/src/api/initApi.ts:33]: That's better) Thank you)

**Review by aderrod94 — COMMENTED** [review]
less code it's a good)

File: website/frontend/src/components/blocks/Header/Header.tsx (hunk 2 above)
Comment (aderrod94) [reply website/frontend/src/components/blocks/Header/Header.tsx]: done)

**Review by osipov-mit — APPROVED** [review]
Good job

**Review by EugenWay — APPROVED** [review]

**Comment by EugenWay** [comment]
Looks nice!

**Pull request merged**